www/src/pages/docs/index.mdx

```diff
@@ -0,0 +1,65 @@
+import { Link } from "gatsby"
+import { Helmet } from "react-helmet"
+
+import Layout from "../../components/layout"
+import { itemListDocs } from "../../utils/sidebar/item-list"
+import { space } from "../../utils/presets"
+import Container from "../../components/container"
+import EmailCaptureForm from "../../components/email-capture-form"
+import DocSearchContent from "../../components/docsearch-content"
+import FooterLinks from "../../components/shared/footer-links"
+import Breadcrumb from "../../components/docs-breadcrumb"
+import PrevAndNext from "../../components/prev-and-next"
+
+# Gatsby.js Documentation
+
+Gatsby is a blazing fast modern site generator for React.
+
+## Get Started
+
+There are two main ways to get started with Gatsby:
+
+1. <Link to="/tutorial/">Tutorial</Link>: Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though these learning resources have helped developers of all skill levels.
+2. <Link to="/docs/quick-start">Quick start</Link>: One page summary of how to install Gatsby and start a new project.
+
+## Go further
+
+1. <Link to="/docs/recipes/">Recipes</Link>: Find some quick answers for how to accomplish some specific, common tasks with Gatsby.
+2. Choose your own adventure and peruse the various sections of the Gatsby docs:
+
+   - <Link to="/docs/guides/">Reference Guides</Link>: Learn about the many different topics around building with Gatsby, like sourcing data, deployment, and more.
+   - <Link to="/docs/api-reference/">Gatsby API Reference</Link>: Learn more about Gatsby APIs and configuration.
+   - <Link to="/docs/releases-and-migration/"> Releases & Migration</Link>: Find release notes and guides for migration between major versions.
+   - <Link to="/docs/conceptual-guide/">Conceptual Guide</Link>: Read high-level overviews of the Gatsby approach.
+   - <Link to="/docs/gatsby-internals/">Gatsby Internals</Link>: Dig into how Gatsby works behind the scenes.
+   - <Link to="/docs/using-gatsby-professionally/"> Using Gatsby Professionally</Link>: Learn tips and tricks for how to explain Gatsby to others at work, so that you have more opportunities to work with Gatsby professionally.
+
+3. Check out the <Link to="/ecosystem/">Ecosystem</Link> libraries for Gatsby starters and plugins, as well as external community resources.
+
+## Start contributing
+
+Visit the <Link to="/contributing">Contributing</Link> section to find guides on the Gatsby community, code of conduct, and how to get started contributing to Gatsby.
+
+export default ({ children, location }) => (
```

Comment:
You shouldn't need to do all this: the only thing component you'd need is `<EmailCaptureForm>`. Look at `template-doc-markdown` -- the content here is mostly the same as that, except for the email submission form.

Reply:
Cheers, i'll have a look over the weekend. Thanks for the feedback

Reply:
Hi @tesseralis I've fixed up the pull request. I still wasn't sure the best way to set up the email capture form. What's your thoughts on how i've achieved this?